src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,11 @@
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+        for (int i = 0; i < users.length; i++) {
+            if (email.equals(users[i].getEmail())) {
```

Comment:
We can get NullPointerException if email == null
```suggestion
            if (users[i].getEmail().equals(email)) {
```

Reply:
Resolve